### ISSUE TITLE
Check if there is any way to also capture additional issues in the dashboard

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/dashboard#10]
We currently don't capture these in the table. 

For example, `{effectsize}` is OK in dashboard, but has error in one of the additional checks.

cf. https://cran.r-project.org/web/checks/check_results_effectsize.html

<img width="607" alt="Screenshot 2022-10-25 at 20 24 34" src="https://user-images.githubusercontent.com/11330453/197857164-12841c5d-21a4-4e09-bc6d-e87c399958fb.png">

**Comment by etiennebacher** [comment]
Probably possible to add them. Is this kind of issue a reason for CRAN to archive the package? I'm asking because I don't know if these errors should be labelled "Error" in red in the dashboard or if it's less important

**Comment by IndrajeetPatil** [comment]
Yes, packages can be archived for failing to fix the additional issues.

**Comment by lilpomm** [comment]
Cerrar
